backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/predicate/impl/AbstractLuceneFieldPredicateBuilderFactory.java

```diff
@@ -69,4 +92,13 @@ public LuceneSimpleQueryStringPredicateBuilderFieldContext createSimpleQueryStri
 				EventContexts.fromIndexFieldAbsolutePath( absoluteFieldPath )
 		);
 	}
+
+	@Override
+	public ExistsPredicateBuilder<LuceneSearchPredicateBuilder> createExistsPredicateBuilder(String absoluteFieldPath) {
+		return new LuceneExistsPredicateBuilder( absoluteFieldPath, getCodec() );
+	}
+
+	protected abstract LuceneFieldCodec<?> getCodec();
```

Comment:
I think that's a good idea. Why do not move `codec` and `converter` field in there too?

Reply:
Because subtypes require the codec and converter to have a narrower type, so it's easier to just let them declare a field it with the right type.